blog/content/post/2019-05-15-building-bareflank.md

```diff
@@ -0,0 +1,345 @@
+---
+post_title: Building and running Bareflank
+author: krystian.hebel
+layout: post
+published: true
+date: 2019-05-15
+
+tags:
+    - hypervisor
+    - virtualization
+    - bare-metal
+    - VMX
+categories:
+    - Firmware
```

Comment:
@krystian-hebel this is not most recent header, also lack of some preview picture

Reply:
@krystian-hebel  header template with new meta-data is located in `blog/content/post/YYYY-MM-DD*`

Reply:
Done. Cover file from the first post will be reused.

Reply:
Looks fine ti me.